R/utils.R

```diff
@@ -385,6 +386,11 @@
             "Therefore, it is converted to TreeSummarizedExperiment.",
             call. = FALSE)
     }
+    if( !identical(rownames(as.matrix(values)), colnames(x)) ){
+        stop("Samples of the dissimilarity matrix should be the same as the",
+             " samples in columns of the assay specified with 'assay.type'.",
+             " The result is not added to reducedDim.")
+    }
```

Comment:
This message is not true, since the matrix is not usually dissimilarity matrix (it is ordination matrix usually) -->

"Rownames of the matrix should match with colnames(x)."